server/api/batch_inventory.py

```diff
@@ -455,6 +473,129 @@ def validate_choice_name_and_get_choice_id(choice_name: str) -> Optional[str]:
         )
         batch_inventory_data.election_results = election_results
 
+    def process_ess_md():
+        """Process CVR files from an older version of ES&S that's currently in use in Maryland.
+        These files do not have a "counting group", and so are given to us as a ZIP file
+        with 5 CSV snapshot files for the ends of the various stages of counting:
+
+        1. Early Voting (EV.csv)
+        2. Election Day (ED.csv)
+        3. Mail-in Ballots #1 (MIB1.csv)
+        4. Mail-in Ballots #2 (MIB2.csv)
+        5. Provisional Ballots (Prov.csv)
+
+        Each file is expected to contain the same rows as all the files above it in the same
+        order, with zero or more rows added compared to the previous file.
+        """
+        ballot_count_by_batch: Dict[BatchKey, int] = defaultdict(int)
+        batch_to_counting_group: Dict[BatchKey, str] = {}
+        batch_tallies: Dict[BatchKey, Dict[str, int]] = defaultdict(
+            lambda: defaultdict(int)
+        )
+
+        def validate_choice_name_and_get_choice_id(choice_name: str) -> Optional[str]:
```

Comment:
Maybe move this to a helper that can be used by both `process_ess` and `process_ess_md`